task-launcher/src/tasks/intro/trials/instructions.ts

```diff
@@ -38,13 +38,14 @@ export const instructions = instructionData.map(data => {
                     <button id="replay-btn-revisited" class="replay">
                         ${replayButtonSvg}
                     </button>
-                    <div class="lev-row-container instruction">
-                        <p>${t[data.prompt]}</p>
+                    <div class="lev-row-container instruction" style="width: 90vw;">
+                        <p style="font-size: 1.5em;">${t[data.prompt]}</p>
```

Comment:
I would prefer if these went into the class definitions. You can add a new definition of font size called:
`font-size-xs-em: 1.5em` and then update the:
```
&.instruction {
    display: flex;
    padding: $lev-spacing-s;
    justify-content: center;
    align-items: center;
    text-align: center;
    border: 3px solid $bg-accent_secondary;
    border-radius: 32px;
    background-color: $bg-bubble-base;
   width: 90vw;

    p {
      font-size: $font-size-xs-em;
      font-weight: $font-weight-light;
      line-height: 1.0;
      margin: 0;
    }
```
Does the width need to be 90vw? or should the max-width be 90vw? We want consistent experience so we will need to just make sure it looks good across different tasks.

Reply:
That makes sense, I'll work on adding it to the class definition. I think setting max-width to 90vw would probably be better, so I'll try that - thanks!

Reply:
I ended up making an `instructions-small` class with the font size and max-width changes and using that for intro - I thought making the font smaller for all tasks didn't look as good. Let me know if you would prefer a different design though!